config/crds/sources_v1alpha1_edgesource.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: edgesources.sources.eventing.knative.dev
+spec:
+  group: sources.eventing.knative.dev
+  names:
+    kind: EdgeSource
+    plural: edgesources
```

Comment:
`short: edge` ?

Reply:
I think short names should be added very carefully.